src/components/ProductCard.tsx

```diff
@@ -16,82 +17,57 @@ export default function ProductCard({ content }: { content: Content }) {
-              className="group-hover:visible flex flex-row absolute top-0 right-0 py-1 px-2 text-xs bg-gray-300 z-20 "
-              onClick={toggleIsFlippedCard}
-            >
-              SHOW MORE
-              <Image
-                className="flex relative left-1"
-                src="/icons8-plus-50.png"
-                alt="X"
-                width={14}
-                height={14}
-                style={{ height: "15px", width: "15px" }}
-              />
-            </button>
-          </Transition>
-          <Transition
-            show={isCardFlipped}
-            className=""
-            enter="transform transition ease-in-out duration-300"
-            enterFrom="opacity-0 -translate-x-0"
-            enterTo="opacity-100 translate-x-0"
-            leave="transform transition ease-in-out duration-300"
-            leaveFrom="opacity-0 translate-x-0"
-            leaveTo="opacity-100 translate-x-0"
+    <div className=" group flex flex-col flex-wrap justify-center border-4 bg-gray-100 border-white">
+      <div className="flex flex-row relative justify-center">
+        <ProductCardShowMoreButton
+          isCardFlipped={isCardFlipped}
+          changeIsCardFlipped={changeIsCardFlipped}
+          changeImage={changeImage}
+          content={content}
+        />
+      </div>
+      <div className="flex flex-col w-[413.33px] h-[508px] items-center relative bg-gray-100">
+        <div className="top-10 relative">
+          <div
+            className={`${
+              isCardFlipped
+                ? "[transform:-rotateY(180deg)]"
+                : "[transform:rotateY(180deg)]"
+            } ease-in-out [transform-style-:preserve-3d] [backface-visblity:hidden] transition-all duration-1000`}
           >
-            <button
-              className="group-hover:visible flex flex-row absolute top-0 right-0 py-1 px-2 text-xs bg-gray-300 z-20"
-              onClick={toggleIsFlippedCard}
-            >
-              CLOSE
-              <Image
-                className="rotate-45 flex relative left-1"
-                src="/icons8-plus-50.png"
-                alt="X"
-                width={14}
-                height={14}
-                quality={100}
-              />
-            </button>
-          </Transition>
-        </div>
-
-        <div className="flex flex-col w-[413.33px] h-[508px] items-center relative">
-          <div className="top-10 relative">
-            <Image src={cardImage} height={300} width={300} alt="cardImage" />
-            <ProductColourButton
-              content={content}
-              selectedColour={selectedColour}
-              setSelectedColour={setSelectedColour}
-              setCardImage={setCardImage}
+            <Image
+              className={`transition-all ease-in-out duration-300`}
+              src={cardImage}
+              height={300}
+              width={300}
+              alt="cardImage"
+              style={{ opacity: opacity / 1 }}
```

Comment:
opacity seems to be 0 or 1, I don't think you need to / 1

Reply:
addressed.